jdaviz/configs/cubeviz/helper.py

```diff
@@ -1,12 +1,18 @@
 import numpy as np
+from packaging.version import Version
 
 from glue.core import BaseData
+from glue_jupyter import __version__ as gj_version
 from jdaviz.core.helpers import ImageConfigHelper
 from jdaviz.configs.default.plugins.line_lists.line_list_mixin import LineListMixin
 from jdaviz.configs.specviz import Specviz
 from jdaviz.core.events import (AddDataMessage,
                                 SliceSelectSliceMessage)
 
+# NOTE: this and the if-statement that uses it can be removed if/once
+# the version of glue-jupyter with as_steps support is pinned as min-version
+GLUEJUPYTER_HAS_AS_STEPS = Version(gj_version) >= Version('1.3.0')
```

Comment:
to test this, install https://github.com/glue-viz/glue-jupyter/pull/309 locally and change `GLUEJUPYTER_HAS_AS_STEPS` to `True` (since the version check won't actually resolve to True until 309 is merged and released).